reports/Washington.md

```diff
@@ -149,6 +149,15 @@ id: wa-seattle-12
 * https://www.reddit.com/r/2020PoliceBrutality/comments/gycscp/cant_go_1_day_without_teargaslighting_us/?utm_source=share&utm_medium=web2x
 * https://www.forbes.com/sites/jemimamcevoy/2020/06/08/seattle-police-use-tear-gas-against-protestors-despite-city-ban/#7e98a1d5b4bc
 
+### Woman hit with police projectile resumes breathing after CPR, delivered to hospital | June 7th
+
+Medics attempt to treat "a young woman in her 20s who had taken a police projectile to her chest" as police throw flash bang grenades directly into the first aid center. There were several cycles of the victim no longer responding or breathing but becoming responsive again after CPR. 911 could not send an ambulance. Eventually a civilian transported her by car to the ED where she was delivered with a pulse.
```

Comment:
Am I missing something related to hospitals or should `ED` be `ER`

Reply:
Some ERs are apparently called Emergency Departments. The firsthand account used ED so I followed that.